shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm

```diff
@@ -35,6 +39,16 @@
 }
 
 void VsyncWaiterIOS::AwaitVSync() {
+  double new_max_refresh_rate = [DisplayLinkManager displayRefreshRate];
+  if (fml::TaskRunnerChecker::RunsOnTheSameThread(
+          task_runners_.GetRasterTaskRunner()->GetTaskQueueId(),
+          task_runners_.GetPlatformTaskRunner()->GetTaskQueueId())) {
+    new_max_refresh_rate = 80;
```

Comment:
I originally set it to 60, but it looks like we can def hit 80 consistently. I decided to lock it at 80 so the scrolling with PlatformView is still smoother than 60fps.

This might change when we re-evaluate what the default fps we want the Flutter apps to run on after we implement a new API for the APP to set refresh rates, see https://github.com/flutter/flutter/issues/119268

Reply:
Can you add a comment for why you chose this?  What hardware did you try?